src/pages/Main/Main.js

```diff
@@ -1,10 +1,34 @@
 import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearch } from '@fortawesome/free-solid-svg-icons';
+import ShopListContainer from './ShopListContainer/ShopListContainer';
+import EatDealContainer from './EatDealContainer/EatDealContainer';
+import './Main.scss';
 
 class Main extends React.Component {
   render() {
     return (
       <div className="Main">
-        <div>Main 입니다</div>
+        <header className="MainHeader">
+          <div className="MainTitle">
```

Comment:
- 소문자로 시작해주세요! 대문자로 시작하는 건 컴포넌트를 네이밍하실 때만 입니다!
```suggestion
<div className='main'>
  <header className='mainHeader'>
    <div className='mainTitle'>
```

Reply:
넵 수정했습니다👍